plugins/woocommerce/src/Internal/BatchProcessing/BatchProcessingController.php

```diff
@@ -274,7 +274,12 @@ public function is_scheduled( string $processor_class_name ): bool {
 	 * @throws \Exception If it's not possible to get an instance of the class.
 	 */
 	private function get_processor_instance( string $processor_class_name ) : BatchProcessorInterface {
-		$processor = wc_get_container()->get( $processor_class_name );
+
+		try {
+			$processor = wc_get_container()->get( $processor_class_name );
+		} catch ( \Exception $e) {
+			$processor = null;
+		}
```

Comment:
We can avoid the exception altogether (and still get an useful `null`) if we use the container's `has` method instead:

```suggestion
		$container = $wc_get_container();
		$processor = $container->has( $processor_class_name ) ? $container->get( $processor_class_name ) : null;
```

Reply:
Thanks for working on it, I agree with @Konamiman's suggestion here, if we can prevent exception all together by using `$wc_get_container()->has` method, that would be great

Reply:
Just to confirm you do mean `wc_get_container()` and _not_ `$wc_get_container()` right? Commit incoming...

Reply:
Indeed, silly typo, sorry